outputs.tf

```diff
@@ -1,15 +1,15 @@
 output "table_name" {
-  value       = "${aws_dynamodb_table.default.name}"
+  value       = "${element(concat(aws_dynamodb_table.default.*.name, list("")), 0)}"
```

Comment:
maybe `splat+join` is shorter?

value       = "${join("", aws_dynamodb_table.default.*.name)}"

Reply:
It's shorter, but less elegant imho, CP is also using it here. Shall I change it ? https://github.com/cloudposse/terraform-aws-iam-user/blob/master/outputs.tf